src/sage/interfaces/magma_free.py

```diff
@@ -26,8 +26,9 @@ def magma_free_eval(code, strip=True, columns=0):
     Use the free online MAGMA calculator to evaluate the given
     input code and return the answer as a string.
 
-    LIMITATIONS: The code must evaluate in at most 20 seconds
-    and there is a limitation on the amount of RAM.
+    .. WARNING::
```

Comment:
```suggestion
    .. WARNING::

```

I think @tscrim's suggestion was to remove the spaces, not the blank line.

Reply:
Yes, that’s right. GH seems to have thought I meant to remove the entire line…but that’s not what was intended.